.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -890,6 +920,18 @@ private Regex BuildTextNumberRegex()
                                     GetKeyRegex(this.Config.CardinalNumberMap.Keys);
             }
 
+            // For Hindi, there is a need for another NumberMap of the type double to handle values like 1.5.
+            // As this cannot be included into either Cardinal or Ordinal NumberMap as they are of the type long,
+            // HindiDecimalUnitsList (type double) takes care of these entries and it needs to be added to the singleIntFrac
+            // for the extractor to extract them
+            if (this.Config.CultureInfo.Name == "hi-IN")

```

Comment:
Language-specific initialization is acceptable. But it would be preferable if this could be a method in config.
As Italian already added this workaround, no need to change Hindi only at this stage though.